RELEASE_PROCESS.md

```diff
@@ -30,38 +30,86 @@ In order to release, you will need the following accounts/permissions:
 ### Informing everyone of a pending release
 
 - [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
+CLARIFICATION from @tas:
+    we can probably move the cft announce to be post promote
+    just copy the discourse post link to that channel to make sure folks see it
+    TODO: get the discourse link he was referring to.
```

Comment:
Announce the intention for release in #a2-release-coordinate channel. The link to discourse post can be found after release like: https://discourse.chef.io/t/chef-infra-server-14-1-0-released/19616

Reply:
What about the other two channels - #cft-announce and #chef-server ?